docs/introduction/index.rst

```diff
@@ -159,3 +159,37 @@ The following backends can be used with Flocker:
 Configuration details for each of the backends can be found in the :ref:`Configuring the Nodes and Storage Backends<agent-yml>` topic.
 
 .. XXX FLOC 3144 - add a link here to the instructions on how to create your own backend driver
+
+.. _get-started:
+
+Get Started with Flocker
+========================
+
+Before you begin to install Flocker, there are a couple of ways in which you can try out Flocker without having to go through the full installation steps:
+
+Flocker in a Live Hosted Environment
+------------------------------------
+
+You can take Flocker for a spin using a free, live hosted environment. 
+No installation is required, and it’s great for getting a hands-on introduction to what Flocker is all about.
+
+* The live demo environment is hosted by ClusterHQ.
+* The demo environment will include a fully-installed and configured 2-node Flocker cluster and CLI.
+* You will have root access to this environment, so you can test Flocker however you want.
+* Follow a step-by-step tutorial.
+* To give you time to play, you’ll have access to the environment for 3 hours.
+
+For more information, see the `Try Flocker`_ page.
+
+.. _vagrant-install:
+
+Flocker Installed Locally Using Vagrant
+---------------------------------------
+
+If you don’t want to use real servers, you can set up a cluster locally on virtual machines using our Vagrant image.
```

Comment:
Out of its previous context, the clause "If you don't want to use real servers" seems wrong. Perhaps "For quick experiments, you can set up..."